### DISCUSSION TITLE
Schema for REST /imp

### DISCUSSION THREAD

**Discussion by robocoder** [questdb/questdb#1303] [Ideas]
How does one specify that the column has an index? Something along the lines of
```
 {"name":"currency", "type":"SYMBOL", "index":true},
```

**Comment by bsmth** [comment]
Hi @robocoder, thanks a lot for the discussion -> I've opened an issue at https://github.com/questdb/questdb/issues/1345

**Comment by marregui** [comment]
@robocoder in principle that would be the way, but at the moment it is not supported. I am creating a PR today to support it. This can be followed in the issue #1345.
